src/backend/stream.rs

```diff
@@ -299,7 +299,7 @@ impl<'ctx> PulseStream<'ctx> {
         fn check_error(s: &pulse::Stream, u: *mut c_void) {
             let stm = unsafe { &mut *(u as *mut PulseStream) };
             if !s.get_state().is_good() {
-                cubeb_log!("Calling error callback");
+                cubeb_alog!("Calling error callback");
```

Comment:
Changed from `cubeb_log` -> `cubeb_alog` here since we're in `check_error` -- can revert as needed